release/gcb/gcb_lib.sh

```diff
@@ -134,12 +134,15 @@ function make_istio() {
   # long list of space-delimited filepaths --- and each artifact would get
   # injected into the Docker image.
   go run tools/license/get_dep_licenses.go --branch "${BRANCH}" > LICENSES.txt
+  # Get hash of current with revision (e.g. c2bc0d223739c841c5e810c6c439562aa9d67f5f-dirty)
+  git describe --dirty --match=NeverMatch --always --abbrev=40 > COMMIT_HASH.txt
```

Comment:
will this be a new file included in the release?

Reply:
That is the idea. We wanted to install `master-latest-daily` on a regular basis, but then we realised that there are no helm repos published for the daily releases. So we wanted to create the `istio.yaml` from the code. But the daily releases are not tagged as far as we checked (interested choice by the way). So we found no way to determine which code is exactly in a `master-daily-latest` release.
In this case, the file `COMMIT_HASH.txt` would be handy.

Or are we overlooking something? Is there a way to determine which code is within `master-latest-daily`? Other than making a good guess based on the timestamp in the other tag attached to those images :)

Reply:
does something like this work? https://storage.googleapis.com/istio-prerelease/daily-build/master-latest-daily/manifest.txt

Reply:
Thanks. Yes, that should do and is of course simpler. We will adapt the PR on Monday. Do you know if this file in also written for local builds? But we could of course we could just tag our fork or save it otherwise in that case. Just out of curiosity.

Reply:
not sure where it's created, if needed we may be able to move it's creation to the local release step. All the local release stuff is brand new so there may be some things that aren't done for the local builds that we could do